dev/com.ibm.ws.ejbcontainer.security.jacc_fat/fat/src/com/ibm/ws/ejbcontainer/security/jacc_fat/audit/PureAnnA08JAASLoginFromEJBJACCAuditTest.java

```diff
@@ -49,6 +56,12 @@
 public class PureAnnA08JAASLoginFromEJBJACCAuditTest extends EJBAnnTestBase {
 
     protected static Class<?> logClass = PureAnnA08JAASLoginFromEJBJACCAuditTest.class;
+    @ClassRule
+    public static RepeatTests r = EERepeatActions.repeat(Constants.SERVER_EJB_AUDIT,
+    EERepeatActions.EE10, // EE10
+    EERepeatActions.EE9, // EE9
+    EERepeatActions.EE8).with(new EmptyAction().fullFATOnly())
+    .andWith(new FeatureReplacementAction(AuditCommonTest.AUDIT_1_0_FEATURES, AuditCommonTest.AUDIT_2_0_FEATURES));
```

Comment:
```suggestion
    /**
     * Need the first repeat to make sure that audit-2.0 from a previous repeat gets put back to audit-1.0
     */
    @ClassRule
    public static RepeatTests auditRepeat = RepeatTests.with(new FeatureReplacementAction("audit-2.0", "audit-1.0").fullFATOnly())
                    .andWith(new FeatureReplacementAction("audit-1.0", "audit-2.0"));
```
Only need the audit repeat because the Jakarta repeat is already happening in FATSuite.